setup.py

```diff
@@ -43,6 +43,7 @@
 Programming Language :: Python :: 3
 Programming Language :: Python :: 3.8
 Programming Language :: Python :: 3.9
+Programming Language :: Python :: 3.10
```

Comment:
to be clear, this is the only line we want to backport as far as I can tell from checking the same files on `maintenance/1.8.x`